notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>I think we need to provide link on NNCf and PTQ API https://docs.openvino.ai/latest/basic_qauntization_flow.html#doxid-basic-qauntization-flow</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>done!</p>